client/my-sites/stats/site.jsx

```diff
@@ -110,56 +107,19 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var site = this.props.sites.getSite( this.props.siteId ),
-			charts = this.props.charts(),
-			queryDate = this.props.date.format( 'YYYY-MM-DD' ),
-			oldStatsLink = 'https://wordpress.com/my-stats/?poll=1&blog=' + this.props.siteId,
-			currentUser = user.get(),
-			period = this.props.period.period,
-			moduleStrings = statsStrings(),
-			oldStatsMessage = null,
-			tagsList,
-			nonPeriodicModules,
-			videoList;
+		const site = this.props.sites.getSite( this.props.siteId );
+		const charts = this.props.charts();
+		const queryDate = this.props.date.format( 'YYYY-MM-DD' );
+		const period = this.props.period.period;
+		const moduleStrings = statsStrings();
+		let nonPeriodicModules;
+		let videoList;
 
 		debug( 'Rendering site stats component', this.props );
 
-		if ( ! site.jetpack && 'wp-admin' === store.get( 'oldStatsLink' ) ) {
```

Comment:
Please verify this (a second set of eyes always helps), but from my inspection of the original code, the code was no longer actually used. The `oldStatsMessage` variable was never included in any output (hence also making the `oldStatsLink` obsolete). This code appears to be obsolete code that was never cleaned up. I removed it to avoid ESLint warnings about unused variables.